assets/js/admin/admin.js

```diff
@@ -221,4 +221,61 @@
 		$(this).append(video);
 
 	});
+
+	// Toggles edit state.
+	function toggleEditTitle( e ) {
+		e.preventDefault();
+
+		var $ele = $( '#edit-form-name' ),
+			$input_title = $ele.siblings( '#evf-edit-form-name' );
+		// Toggle disabled property.
+		$input_title.prop( 'disabled' , function(_, val) {
+			return ! val;
+		});
+		if ( ! $input_title.hasClass( 'everst-forms-name-editing' ) ) {
+			$input_title.focus();
+		}
+		$input_title.toggleClass( 'everst-forms-name-editing' );
+	}
+
+	// Delegates event to toggleEditTitle() on clicking
```

Comment:
Docblock should always end with a full stop as it sounds incomplete with it.

Reply:
I've done it as per wp standards :D